main.go

```diff
@@ -30,14 +32,23 @@ func init() {
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
+	var watchInterval time.Duration
+	var prometheusURL string
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
```

Comment:
I think `pvc-autoresizer` should always elect the leader.
Please drop this flag.